dvc/command/version.py

```diff
@@ -115,10 +111,12 @@ def get_linktype_support_info(repo):
                 os.unlink(dst)
             except DvcException:
                 status = "not supported"
-            cache.append(f"{name} - {status}")
+
+            if status == "supported":
+                cache.append(f"{name}")
```

Comment:
No need of f-strings here.
```suggestion
                cache.append(name)
```